src/templates/layout.swig

```diff
@@ -116,9 +119,84 @@
 				});
 			})
 			.queueScript('/scripts/app.min.js')
-			.queueScript('https://cdn.segment.com/analytics.js/v1/VtTlzxOC5QJ1H0zermvo84cAbReTofDU/analytics.min.js')
+			.queueWait(function () {
+				/*
+				 * Segment Analytics Preload
```

Comment:
Huh – I'm surprised that this wasn't anywhere already, which seems like a big oversight (I mean, most events will still be captured, but it does mean we're probably missing a bunch of initial pageview events...)